src/main/spark2.3.2/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedColumnReader.java

```
@@ -532,6 +532,7 @@ private void readFixedLenByteArrayBatch(
 
   protected void readPage() {
     DataPage page = pageReader.readPage();
+    if(null == page) return;
```

if (page == null) is true I think we should throw a IOE instead of silent return, `readPage` not a public method, if there is not page, the invocation logic is wrong already